src/main/kotlin/agallipo/BoardingCard.kt

```diff
@@ -1,59 +1,59 @@
-package agallipo
+    package agallipo
 
-import java.io.File
-import java.io.InputStream
+    import java.io.File
+    import java.io.InputStream
+    //created by alex :)
+    data class BoardingCard(var origin: String?, var destination: String?,  var transport: String?, var gate: String?, var seat: String?)
```

Comment:
All members of this class are nullable. I don't know what are the specifications of the problem, but it seems reasonable that fields of a boarding card like "origin" or "destination" cannot be null.